sdk/py/apepay/__init__.py

```diff
@@ -27,10 +27,9 @@
     TokenNotAccepted,
     ValidatorFailed,
 )
-from .settings import Settings
 from .utils import time_unit_to_timedelta
 
-settings = Settings()
```

Comment:
I also think settings can likely be removed, shouldn't be a part of the SDK

Reply:
I would prefer to only handle the bug in question in this PR.